app/views/govuk_publishing_components/components/_checkboxes.html.erb

```diff
@@ -34,30 +34,34 @@
       <%= tag.span error, id: "#{id}-error", class: "govuk-error-message" %>
     <% end %>
 
-    <%= tag.div class: "govuk-checkboxes", data: {
+    <%= tag.ul class: "govuk-checkboxes gem-c-checkboxes__list", data: {
       module: ('checkboxes' if has_conditional),
       nested: ('true' if has_nested),
     } do %>
       <% items.each_with_index do |item, index| %>
-        <%= render 'govuk_publishing_components/components/checkbox', {
-          id: id,
-          name: name,
-          item: item,
-          index: index,
-        } %>
-        <% if item[:conditional] %>
-          <%= tag.div item[:conditional], id: "#{id}-conditional-#{index}", class: "govuk-checkboxes__conditional govuk-checkboxes__conditional--hidden" %>
-        <% end %>
+        <%= tag.li class: "gem-c-checkboxes__list-item" do %>
+          <%= render 'govuk_publishing_components/components/checkbox', {
+            id: id,
+            name: name,
+            item: item,
+            index: index,
```

Comment:
Is it possible to pass data attributes to these checkboxes? And same for the nested ones.

(I realise this was already in so maybe something to consider separately)

Reply:
I think it's best to consider separately, do you have a need right now?

Reply:
Not urgently, no. We can add it in when we need it.